themes/helphub/sass/modules/_table-of-contents.scss

```diff
@@ -1,52 +0,0 @@
-.table-of-contents {
-	background-color: #f5f5f5;
-	float: right;
-	max-width: 35%;
-	border-top: 5px solid #d0d0d0;
-	padding-top: 2%;
-	padding-bottom: 2%;
-
-	ul {
-		margin-left: 0.2em;
-	}
-
-	li {
-		list-style: none;
-		padding-left: 5%;
-		padding-right: 3%;
-		padding-bottom: 2%;
-		font-weight: 900;
-		line-height: 1.2;
-	}
-}
-
-.single-post {
-
-	h1 {
-
-		&.entry-title {
-			font-family: HelveticaNeue-Light, "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, sans-serif;
-			font-weight: 300;
-			font-size: 2.5em;
-		}
-	}
-}
-
-h2,
-h3,
-h4 {
-
-	&.toc-heading {
-		margin-top: 2em;
-		font-size: 1.2em;
-		font-weight: 400;
-	}
-}
-
-p {
-
-	&.toc-jump {
-		float: right;
-		font-size: 1em;
-	}
-}
```

Comment:
With this whole file being removed it should also be removed fro import in `style.scss`:

https://github.com/Kenshino/HelpHub/blob/master/themes/helphub/sass/style.scss#L97

Reply:
It is removed.

https://github.com/Kenshino/HelpHub/pull/146/files#diff-ae43b438d537cbe2c607d5f83ed69e79

Reply:
Ha, I looked twice, my bad, must still be in holiday mode ;)